doc/asciidoc/procedures/index.adoc

```diff
@@ -241,3 +241,64 @@ Supported deserializers are: `org.apache.kafka.common.serialization.ByteArrayDes
 
 |===
 
+=== Streams Sink Lifecycle procedure
```

Comment:
can we add a usage example of an (textual) output of such a lifecycle?

```
listconfig
stop
status
start
status
restart
status on non leader
```

Reply:
we can also generate that from a test an just adoc-include the file.

Reply:
done